client/src/js/services/PrototypeApiService.js

```diff
@@ -161,4 +163,32 @@ function PrototypeApiService($http, util) {
     return this.$http.delete(target)
       .then(util.unwrapHttpResponse);
   }
+
+  /**
+   * @method reports
+   *
+   * @description
+   * Sends an HTTP GET request to the url "/route/reports/:id"
```

Comment:
Ah, I understand.. `/route` here is the base URL.  That makes sense.  Thanks for clearing that up!